### ISSUE TITLE
Repo assessment level 2

### ISSUE THREAD

**Issue by luxiya01** [jadlers/clear-the-field#35]
The project owner has very good committing habit. The commit messages are short, clear and informative. Most of the commit messages refer to some of the issues, making it easy to track the purpose of this commit. Descriptive images are used to show whether one commit results in a fail or success of a test, which is really good! 

The only small drawback that I can find is that for some commit messages it is slightly unclear what this commit is for, i.e. which class has been fixed. This is especially true for the commits that don't refer to any issues. 

To sum it up, I have learnt a lot from the commit messages of this repo and will try to use this knowledge gained to improve my own committing habit...
